
 15%|█████████████████████▋                                                                                                                            | 25288704/170498071 [00:01<00:06, 23854291.99it/s]
Training with config: {'epochs': 2, 'batch_size': 256, 'learning_rate': 0.0014, 'beta_0': 0.9, 'beta_1': 0.999, 'weight_decay': 0}




170500096it [00:07, 21468242.08it/s]
Files already downloaded and verified
Traceback (most recent call last):
  File "/Users/alexischevalier/Python_projects/MLAB_public/w1d4_answers_part2.py", line 149, in <module>
    train(wandb.config)
  File "/Users/alexischevalier/Python_projects/MLAB_public/w1d4_answers_part2.py", line 76, in train
    trainloader = DataLoader(cifar_train, batch_size=config.batch_size, shuffle=True, pin_memory=True)
AttributeError: 'dict' object has no attribute 'batch_size'